src/test/java/com/loofah/graph/api/queries/SkillQueryTest.java

```diff
@@ -28,12 +28,12 @@
     private SkillQuery skillQuery;
 
     @Before
-    public void setUp() throws Exception {
+    public void setUp() {
         skillQuery = new SkillQuery(skillRepository);
     }
 
     @Test
-    public void get_findSkillFromRepositoryWithIdFromDataFetchingEnvironment() {
+    public void get_findsSkillsFromRepositoryWithGivenId() {
```

Comment:
[suuuper tiny little thing] test name should be "findSkill" (singular) rather than "findSkills"

Reply:
good spot - updated!